client/src/views/devices/ExportRecoveryDevicePage.vue

```diff
@@ -3,100 +3,142 @@
+                </div>
               </div>
-              <div class="block">
-                {{ $t('ExportRecoveryDevicePage.titles.recoveryKey') }}
+              <div class="file-item">
+                <div class="file-item__title subtitles-normal">
+                  <ion-icon :icon="key" />
+                  {{ $t('ExportRecoveryDevicePage.titles.recoveryKey') }}
+                </div>
               </div>
             </div>
-            <ion-button
-              @click="exportDevice()"
-              id="exportDevice"
-            >
-              {{ $t('ExportRecoveryDevicePage.actions.understand') }}
-            </ion-button>
           </div>
-          <div v-else-if="state === ExportDevicePageState.Download">
-            <ms-informative-text>
-              {{ $t('ExportRecoveryDevicePage.subtitles.keepFilesSeparate') }}
-            </ms-informative-text>
 
-            <div>
-              <div class="block">
+          <ion-button
+            @click="exportDevice()"
+            id="exportDevice"
+          >
+            {{ $t('ExportRecoveryDevicePage.actions.understand') }}
+          </ion-button>
+        </div>
+
+        <!-- STEP 2 • Download files -->
+        <div
+          class="recovery-container download"
+          v-else-if="state === ExportDevicePageState.Download"
+        >
+          <ms-informative-text>
+            {{ $t('ExportRecoveryDevicePage.subtitles.keepFilesSeparate') }}
+          </ms-informative-text>
+
+          <div class="file-list">
+            <div class="file-item">
+              <div class="file-item__title subtitles-normal">
+                <ion-icon :icon="document" />
                 {{ $t('ExportRecoveryDevicePage.titles.recoveryFile') }}
-                <div v-show="!recoveryFileDownloaded">
-                  <ion-button
-                    @click="downloadRecoveryFile()"
-                    id="downloadButton"
-                  >
-                    {{ $t('ExportRecoveryDevicePage.actions.download') }}
-                  </ion-button>
-                </div>
-                <div v-show="recoveryFileDownloaded">
-                  <ion-icon
-                    :icon="checkmarkCircle"
-                    class="checked"
-                  />
-                  {{ $t('ExportRecoveryDevicePage.subtitles.fileDownloaded') }}
-                </div>
               </div>
-              <div class="block">
+              <ion-text class="file-item__description body">
+                {{ $t('ExportRecoveryDevicePage.subtitles.fileExplanation') }}
+              </ion-text>
+              <div
+                class="file-item__button"
+                v-show="!recoveryFileDownloaded"
+              >
+                <ion-button
+                  @click="downloadRecoveryFile()"
+                  id="downloadButton"
+                >
+                  {{ $t('ExportRecoveryDevicePage.actions.download') }}
+                </ion-button>
```

Comment:
Missing download icon in front of button text (mockup / current)
<img width="418" alt="Screenshot 2023-12-01 at 15 32 21" src="https://github.com/Scille/parsec-cloud/assets/41511029/824a98f2-ef5d-46da-96f6-51fa4902aa6f">
<img width="390" alt="Screenshot 2023-12-01 at 15 32 26" src="https://github.com/Scille/parsec-cloud/assets/41511029/e68b85f2-caf1-480c-bc40-9473583f4db3">